src/main/java/org/springframework/data/tarantool/core/mapping/BasicTarantoolPersistentEntity.java

```diff
@@ -32,4 +44,82 @@ public String getSpaceName() {
     public boolean hasTupleAnnotation() {
         return getType().getAnnotation(Tuple.class) != null;
     }
+
+    @Override
+    public boolean hasTarantoolIdClassAnnotation() {
+        return getType().getAnnotation(TarantoolIdClass.class) != null;
+    }
+
+    @Override
+    public Class<?> getTarantoolIdClass() {
+        if (!hasTarantoolIdClassAnnotation()) {
+            return null;
```

Comment:
We need to check that this `null` does not break anything. Maybe it's better to return `Optional` here?